src/script/components/Modals/PrimaryModal/Components/MessageContent.tsx

```diff
@@ -1,6 +1,6 @@
 /*
  * Wire
```

Comment:
This makes no sense git!

One file was deleted and another newly created, and git mixed them up :D